tidy3d/web/core/task_core.py

```diff
@@ -472,17 +471,31 @@ def get_sim_data_hdf5(
         if not self.task_id:
             raise WebError("Expected field 'task_id' is unset.")
 
+        try:
+            return download_gz_file(
+                resource_id=self.task_id,
+                remote_filename=SIMULATION_DATA_HDF5_GZ,
+                to_file=to_file,
+                verbose=verbose,
+                progress_callback=progress_callback,
+            )
+        except Exception:
+            # TODO: would be good to have a debug log message here but it's not clear to me how to
+            # use the frontend logger here, or what the `core_config.py` console does
+            pass
+
         try:
```

Comment:
does this code block ever get run? it seems the previous `try: except:` will cause the function to return or error?

Reply:
oh I just noticed the `pass`, my bad. I think a cleaner way to handle what you're doing here is to nest the next try except in the except block. see the accepted answer here: https://stackoverflow.com/questions/17322208/multiple-try-codes-in-one-block

Reply:
It's just a `pass` in the except clause. But see my TODO about how best to log a message, any thoughts?

I could also move this part into the except clause though, the way it is in the mode.web.

Reply:
I think this would be the best way to log

https://github.com/flexcompute/tidy3d/blob/a9004b8c2ad1ebe028ef6c5ed7662d997c48ae65/tidy3d/web/core/task_core.py#L312-L313

but yea in that stackoverflow message, there's some mention that if you are doing this nesting of try/except you might want to refactor, so I'm not sure if it is ultimately the best approach.

Maybe we could just flag this as something to improve when we get around to properly fixing up the webapi though.